### ISSUE TITLE
Terrible. you have to be kidding.

### ISSUE THREAD

**Issue by rpierceneu** [MicrosoftDocs/windows-itpro-docs#11009]
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 789566d4-463a-4c64-bf89-fb963ac29f80
* Version Independent ID: 02946732-bb29-d8e6-cedd-7f5e508f8798
* Content: [Manage Windows Hello in your organization (Windows) - Windows security](https://learn.microsoft.com/en-us/windows/security/identity-protection/hello-for-business/hello-manage-in-organization)
* Content Source: [windows/security/identity-protection/hello-for-business/hello-manage-in-organization.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/identity-protection/hello-for-business/hello-manage-in-organization.md)
* Product: **windows-client**
* Technology: **itpro-security**
* GitHub Login: @paolomatarazzo
* Microsoft Alias: **paoloma**

**Comment by paolomatarazzo** [comment]
Hello @rpierceneu and thank you for the invaluable feedback! Can you please clarify what is "Terrible." so that we can improve the documentation?

**Comment by rpierceneu** [comment]
Hello – Please give me a little more context.  I cannot find an outbound email re: the subject u are referring to.
Thank you

Robert
Robert Pierce Ph.D.  |  Senior Solution Architect
Neudesic<http://www.neudesic.com/>, an IBM Company | The Trusted Technology Partner in Business Innovation
M: 214-405-6547  |  LinkedIn<http://www.linkedin.com/in/robertpiercephd>
[A picture containing text, sign, blue  Description automatically generated][A picture containing text, sign, blue  Description automatically generated]

**Comment by paolomatarazzo** [comment]
@rpierceneu you provided the GitHub feedback: "Terrible. you have to be kidding. #11009" in the ralation to the content: https://learn.microsoft.com/en-us/windows/security/identity-protection/hello-for-business/hello-manage-in-organization
Asking for clarity.

**Comment by rpierceneu** [comment]
Ah Yes. Now I remember. I have a Surface Book Pro 2 and did a bare metal wipe and reinit a weekend ago. The machine is domain joined. I installed a regular Win10Pro 22H2 installer (not the Surface installer – couldn’t figure that one out). After I installed Win10 and started setting things up and joined the domain I noticed that Windows Hello Face did not work even though it worked perfectly before the wipe. I spent 2 days (on and off while doing other stuff) trying to find out why. When I used the local login I could get Hello Face to work fine but no good with the domain login. So I was searching everywhere for this answer and I came across the link below which had no answer and was very lengthy and in my frustration sent you guys the “you gotta be kidding”. 😊  I eventually found the answer after many tries. Fortunately for you, you have contacted me and I can share the answer with you. It is this!! Hope it helps. It takes a village! Take care.

***@***.***

And this
***@***.***



Robert
Robert Pierce Ph.D.  |  Senior Solution Architect
Neudesic<http://www.neudesic.com/>, an IBM Company | The Trusted Technology Partner in Business Innovation
M: 214-405-6547  |  LinkedIn<http://www.linkedin.com/in/robertpiercephd>
[A picture containing text, sign, blue  Description automatically generated][A picture containing text, sign, blue  Description automatically generated]
